portfolio/src/main/webapp/comment.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches content from the server and adds it to the DOM.
+ */
+function getContentFunctions() {
+    const num = document.querySelector('[name="number-comments"]').value;
+    
+    // Check that the input is between 1 and 5.
+    if(num > 5 || num < 0) {
```

Comment:
styling nit: it's pretty common to have a space between parentheses and the preceding keyword
Ex: `if (num > 5 ...`
See examples [here](https://google.github.io/styleguide/jsguide.html#formatting)